map_size_meters: 2100.0
roi_size_meters: 210.0
voxel_size_meters: 0.30
min_hits_for_occupancy: 40
max_hits_in_cell: 200
min_object_seperation_distance: 3.0
ogrid_inflation_cell: 2

executive_rate: 10

number_persistant_point_clouds: 20

filter_points_leaf_size_x: 0.2
filter_points_leaf_size_y: 0.2
filter_points_leaf_size_z: 0.2

icp_max_corespondence_distance: 0.1
icp_max_iterations: 40
icp_transformation_epsilon: 1E-8
icp_euclidean_fitness_epsilon: 1.0

cluster_tolerance_m: 4.4
cluster_min_num_points: 2
cluster_max_num_points: 25000

object_types: ["SHOOTER", "SCAN_THE_CODE"]
object_colors: ["RED", "GREEN"]